src/server.js

```diff
@@ -80,6 +80,21 @@ Server.prototype.writeServerConfig = function () {
   fs.writeFileSync(file, config)
 }
 
+Server.prototype.writeBattleEyeConfig = function (config) {
+  if(this.options.battleEye)
+  {
+    var filePath = path.join(this.options.path, 'battleye/BEServer_x64.cfg')
+    fs.writeFileSync(filePath, config)
+
+    var filePath2 = path.join(this.options.path, 'battleye/BEServer.cfg')
+    fs.writeFileSync(filePath2, config)
```

Comment:
Instead of doing this twice you could make this into a loop
```javascript
var self = this
['BEServer.cfg', 'BEServer_x64.cfg'].forEach(function (filename) {
  var filePath = path.join(self.battleEyeConfigPath(), filename)
  fs.writeFileSync(filePath, config)
})
```